### PR TITLE
Add an option to provide github username to be mentioned 

### PR DESCRIPTION
**Pull request by zoewangg**
## Description
add an option in the changelog entry script to provide github username to be mentioned in the CHANGELOG

Sample changelog

# __2.15.10__ __2010-10-16__
## __AWS CodeCommit__
  - ### Features
    - test

        - Contributed by: @awesomecontributor1 

## __AWS S3__
  - ### Features
    - test
        - Contributed by: @awesomecontributor2

## __AWS SDK for Java v2__
  - ### Features
    - test1

## __Contributors__
Special thanks to the following contributors to this release: 

@awesomecontributor1, @awesomecontributor2.

## License

- [x] I confirm that this pull request can be released under the Apache 2 license

### CODE REVIEW

**Comment by dagnir** [comment]
What do you think about adding a `contributor` property to the JSON itself?

```json
{
  "category": "AWS SDK for Java v2",
  "type": "feature",
  "description": "Implement feature Foo.",
  "contributor": "awesomecontributor"
}
```

That should make changelog generation more flexible; we can do something like 

---

## __AWS SDK for Java v2__
  - ### Features
    - Implement feature Foo.
       - Contributed by: @awesomecontributor 

or event aggregate all contributions in one place:

---

## __Contributors__
Special thanks to the following contributors to this release: @awesomecontributor

Would also allows to do things things like add acknowledge contributors for things that don't necessarily have an associated change, like adding new tests or something.

**Comment by zoewangg** [comment]
@dagnir :+1: Good idea! Love it. I'll update it

File: scripts/new-change [hunk 1]
```diff
@@ -61,6 +61,12 @@ type: {change_type}
 # or "AWS SDK for Java v2" if it's an SDK change to the core, runtime etc
 category: {category}
 
+# Your GitHub user name to be included in the CHANGELOG.
+# Every contribution counts and we would like to recognize
+# your contribution!
+# Leave it empty if you would prefer not to be mentioned.
+github_username: {github_username}
```
Comment (millems) [inline scripts/new-change:68]: Does {github_username} count as empty?

Comment (zoewangg) [reply scripts/new-change]: Not sure what you mean, but it's empty by default.

**Review by dagnir — APPROVED** [review]
:shipit:

**Pull request merged**